tasks/section5.yml

```diff
@@ -311,15 +441,19 @@
       - rule_5.2.8
 
 - name: "SCORED | 5.2.8 | PATCH | Ensure SSH root login is disabled"
-  command: /bin/true
+  lineinfile:
+      state: present
+      dest: /etc/ssh/sshd_config
+      regexp: '^PermitRootLogin'
+      line: 'PermitRootLogin no'
   tags:
       - level1
       - level2
       - patch
       - rule_5.2.8
 
 - name: "SCORED | 5.2.9 | AUDIT | Ensure SSH PermitEmptyPasswords is disabled"
-  command: /bin/true
+  command: grep "^PermitEmptyPasswords no" /etc/ssh/sshd_config
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*